webapp/src/shared/components/ChatBubble/ChatBubble.tsx

```diff
@@ -27,7 +29,11 @@ export default function ChatBubble({
   return (
     <div className={`chat-bubble chat-bubble-${variant}`}>
       <div className="chat-bubble-avatar">
-        {avatar && !isConsecutive && <SmallAvatar {...avatar} />}
+        {avatar && !isConsecutive && (
```

Comment:
Since we're already here changing this what do you think about changing this as well:

```diff
diff --git a/webapp/src/shared/components/ChatBubble/ChatBubble.tsx b/webapp/src/shared/components/ChatBubble/ChatBubble.tsx
index 0ca0a38..c8c9e41 100644
--- a/webapp/src/shared/components/ChatBubble/ChatBubble.tsx
+++ b/webapp/src/shared/components/ChatBubble/ChatBubble.tsx
@@ -41,7 +41,7 @@ export default function ChatBubble({
         <div className={'chat-bubble-text-author'}>
           {name && isFirst && (
             <Text
-              variant={TextVariant.CAPTION}
+              variant={TextVariant.PARAGRAPH}
               color={TextColor.LIGHT}
               weight={TextWeight.MEDIUM}
               children={name}
@@ -50,7 +50,7 @@ export default function ChatBubble({
         </div>
         <div className={`chat-bubble-text chat-bubble-text-${variant}`}>
           <Text
-            variant={TextVariant.CAPTION}
+            variant={TextVariant.PARAGRAPH}
             color={TextColor.LIGHT}
             weight={TextWeight.REGULAR}
             children={text}
```

Reply:
Lgtm :+1:, done